### ISSUE TITLE
Please remove mobkit from the pack

### ISSUE THREAD

**Issue by TheTermos** [t-affeldt/climate#3]
The api is intended to be shared between mods using it, it has been reported that this pack breaks other mods using mobkit if run alongside.
Alternatively if you want to rely on a specific version, please rename all occurrences of the namespace 'mobkit' to some other identifier unique to your pack.

**Comment by Adnan1091** [comment]
Really?I have used this mod along with mobkit(i removed the native one)and even updated the mobkit in this modpack and haven't had any issues.

**Comment by TheTermos** [comment]
This is possible, provided there are no relevant differences between the two versions, still, it's going to cause problems in other cases.

edit: one example here: https://forum.minetest.net/viewtopic.php?p=377776#p377776

**Comment by t-affeldt** [comment]
I suppose that can happen. There are no differences to the normal version in my pack. It's literally just a reference to the normal Github branch of that mod.

The reason I included this is because one of my mods relies on it. People wouldn't read the explanatory text and the modpack would break without them having installed mobkit. Hence, I included mobkit for convenience. I suppose you're describing the opposite problem where people again don't check what they download and end up with multiple copies of the same mod. It is as easy as to disable either one of them. I'm not sure what to do here because it will break for some people either way unless they make sure to read the documentation.
